README.rst

```diff
@@ -0,0 +1,63 @@
+# reana-auth-krb5
+
+Required information and inputs:
+
+
+
+## Building the image
+
+No build arguments needed.
+
+
+===========================
+REANA Environment Kerberos5
+===========================
+
+About
+=====
+
+``reana-auth-krb5`` provides a container image with encapsulated runtime
+execution environment for creating an renewing Kerberos tokens. The container image
+includes no additional logic or libraries, just the bare minimum to support the Kerberos operations.
```

Comment:
Maybe add a notion that this is just a sidecar, perhaps "The image is used as a sidecar alongside user-submitted computational workflows and takes care of accessing restricted resources using Kerberos authentication."